conda-build.yaml

```diff
@@ -0,0 +1,46 @@
+# Generated from https://pypi.org/project/conda-build/
+package:
+  name: conda-build
+  version: 3.26.1
+  epoch: 0
+  description: tools for building conda packages
+  copyright:
+    - license: BSD 3-clause
+  dependencies:
+    runtime:
+      - conda
+      - py3-filelock
+      - py3-requests
+      - python-3
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - conda
+      - py3-hatchling
+      - py3-pip
+      - py3-setuptools
+      - py3-wheel
+      - python-3
+      - wolfi-base
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: c2bfe2b44ad34b06d8930aa4239995367f3eaf12
+      repository: https://github.com/conda/conda-build
+      tag: ${{package.version}}
+
+  - runs: |
+      pip install hatch conda-package-handling
+      hatch build
+      find dist
+      python3 -m installer -d "${{targets.destdir}}" dist/conda_build*.whl
+
+update:
+  enabled: true
+  github:
+    identifier: https://github.com/conda/conda-build
```

Comment:
```suggestion
    identifier: conda/conda-build
```

Reply:
lol, doh! I caught that with the test (see above), that led me to file the bug, which I went to do, but forgot to push!

Also, removed the debug `find` that I also removed.